src/houghtransform.jl

```diff
@@ -116,26 +124,29 @@ Parameters:
 -   `img_phase`    = phase of the gradient image   
 -   `scale`        = relative accumulator resolution factor  
 -   `min_dist`     = minimum distance between detected circle centers  
--   `vote_thres`   = accumulator threshold for circle detection  
--   `min_radius:max_radius`   = circle radius range
+-   `vote_threshold`   = accumulator threshold for circle detection
+-   `radii`        = circle radius range
 
 [`canny`](@ref) and [`phase`](@ref) can be used for obtaining img_edges and img_phase respectively.
 
 # Example
 ```julia
 img = load("circle.png")
 
-img_edges = canny(img, 1, 0.99, 0.97)
+img_edges = canny(img, (Percentile(99), Percentile(97)))
 dx, dy=imgradients(img, KernelFactors.ando5)
 img_phase = phase(dx, dy)
 
-centers, radii=hough_circle_gradient(img_edges, img_phase, 1, 60, 60, 3:50)
+centers, radii=hough_circle_gradient(img_edges, img_phase, 3:50)
```

Comment:
This might yield different results from the original example (`min_dist` and `vote_threshold` of 3 rather than 60). You've checked to make sure they are reasonable?

Reply:
I must confess, I don't know how to run the example. Where can I get "circle.png" from?